api/v1/numaresourcesscheduler_normalize_test.go

```diff
@@ -314,3 +316,60 @@ func TestNUMAResourcesSchedulerSpecNormalize(t *testing.T) {
 		})
 	}
 }
+
+func TestNUMAResourcesSchedulerSpec_PreNormalize(t *testing.T) {
+	schedInformer := defaultSchedulerInformer
+
+	schedInformerCustom := SchedulerInformerShared
+
+	greaterVersion, _ := platform.ParseVersion("4.20.1000")
+	equalVersion, _ := platform.ParseVersion(ActivePodsResourcesSupportSince)
+	lessVersion, _ := platform.ParseVersion("4.20.0")
```

Comment:
here we check per Z stream, which is fine and obviously necessary. I wonder if we need to check per Y stream. I tend to say no. WDYT?